src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -145,6 +145,20 @@
 
     mixin       ("mod_decl_item") = "org.rust.lang.core.psi.impl.mixin.RustModDeclItemImplMixin"
 
+    mixin       ("const_item") = "org.rust.lang.core.psi.impl.mixin.RustConstItemImplMixin"
+
+    mixin       ("extern_decl_item") = "org.rust.lang.core.psi.impl.mixin.RustExternDeclItemImplMixin"
+
+    mixin       ("foreign_fn_item") = "org.rust.lang.core.psi.impl.mixin.RustForeignFnItemImplMixin"
+
+    mixin       ("foreign_mod_item") = "org.rust.lang.core.psi.impl.mixin.RustForeignModItemImplMixin"
+
+    mixin       ("static_item") = "org.rust.lang.core.psi.impl.mixin.RustStaticItemImplMixin"
```

Comment:
This mixins are empty at the moment, but if I don't specify a mixin, GrammarKit generates the following

```
public class RustStaticItemImpl extends item_ implements RustStaticItem {
```